.gitignore

```diff
@@ -1,2 +1,4 @@
 /node_modules
 /*.log
+.idea/
+*.iml
```

Comment:
I don't like global ignores (the fact they're not supported out of the box is telling to me), but I can remove the change, no problem